src/EA.Weee.Web.Tests.Unit/Areas/AatfReturn/Controller/SentOnCreateSiteOperatorControllerTests.cs

```diff
@@ -81,26 +81,57 @@ public async void IndexGet_GivenAction_DefaultViewShouldBeReturned()
         [Fact]
         public async void IndexPost_GivenInvalidViewModel_ApiShouldNotBeCalled()
         {
+            var form = new FormCollection();
             controller.ModelState.AddModelError("error", "error");
             var model = new SentOnCreateSiteOperatorViewModel();
             model.OperatorAddressData = new OperatorAddressData("TEST", "TEST", "TEST", "TEST", "TEST", "TEST", Guid.NewGuid(), "TEST");
-            await controller.Index(model);
+            await controller.Index(model, form);
 
             A.CallTo(() => apiClient.SendAsync(A<string>._, A<EditSentOnAatfSite>._)).MustNotHaveHappened();
         }
 
         [Fact]
         public async void IndexPost_GivenValidViewModel_ApiSendShouldBeCalled()
         {
+            var form = new FormCollection();
             var model = new SentOnCreateSiteOperatorViewModel();
             model.OperatorAddressData = new OperatorAddressData("TEST", "TEST", "TEST", "TEST", "TEST", "TEST", Guid.NewGuid(), "TEST");
             var request = new EditSentOnAatfSite();
 
             A.CallTo(() => requestCreator.ViewModelToRequest(model)).Returns(request);
 
-            await controller.Index(model);
+            await controller.Index(model, form);
 
             A.CallTo(() => apiClient.SendAsync(A<string>._, request)).MustHaveHappened(Repeated.Exactly.Once);
         }
+
+        [Fact]
+        public async void IndexPost_GivenValidViewModel_IsOperatorTheSameAsAATFShouldBeSet()
+        {
+            var form = new FormCollection();
+            var model = new SentOnCreateSiteOperatorViewModel();
+            model.OperatorAddressData = new OperatorAddressData("TEST", "TEST", "TEST", "TEST", "TEST", "TEST", Guid.NewGuid(), "TEST");
+
+            form.Add("IsOperatorTheSameAsAATF", "true");
+
+            await controller.Index(model, form);
+
+            A.CallTo(() => requestCreator.ViewModelToRequest(A<SentOnCreateSiteOperatorViewModel>.That.Matches(m => m.IsOperatorTheSameAsAATF == true))).MustHaveHappened(Repeated.Exactly.Once);
+        }
+
+        [Fact]
+        public async void IndexPost_GivenInvalidViewModel_IsOperatorTheSameAsAATFShouldBeSet()
+        {
+            var form = new FormCollection();
+            controller.ModelState.AddModelError("error", "error");
+            var model = new SentOnCreateSiteOperatorViewModel();
+            model.OperatorAddressData = new OperatorAddressData("TEST", "TEST", "TEST", "TEST", "TEST", "TEST", Guid.NewGuid(), "TEST");
+            model.IsOperatorTheSameAsAATF = true;
```

Comment:
This test should it do the reverse check of IndexPost_GivenValidViewModel_IsOperatorTheSameAsAATFShouldBeSet?

Reply:
Having another look, I'm testing the same code as IndexPost_GivenValidViewModel_IsOperatorTheSameAsAATFShouldBeSet, I just split them out because they take a different route through the Post method.

Will remove.